cryostatio/cryostat3
# feat(graphQL): add missing subqueries/mutations

# Welcome to Cryostat3! 👋
## Before contributing, make sure you have:

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Based on: #315 
Fixes #322

## Description of the change:
*This change allows an environment variable to be configured so that...*

## Motivation for the change:
*This change is helpful because users may want to...*

## How to manually test:
1.  `cd src/main/webui`
2. checkout to correspondig [PR](https://github.com/cryostatio/cryostat-web/pull/1227) `gh pr checkout 1227` or `git checkout + PR`
3. ` cd -`
4. `./mvnw clean package ; podman image prune -f ; ./smoketest.bash -O`
5.  Create a new recording or using the existing recordings try to add new labels, edit the labels and delete/remove and verify all work as expected.
6. Archive the recordings and try the same, add, edit and delete the labels and see they work as expected.
